src/main/drivers/pwm_output_stm32f4xx.c

```diff
@@ -167,14 +167,14 @@ void pwmDigitalMotorHardwareConfig(const timerHardware_t *timerHardware, uint8_t
 
     TIM_OCStructInit(&TIM_OCInitStructure);
     TIM_OCInitStructure.TIM_OCMode = TIM_OCMode_PWM1;
-    if (timerHardware->output & TIMER_OUTPUT_N_CHANNEL) {
+    if (output & TIMER_OUTPUT_N_CHANNEL) {
```

Comment:
Most targets use normal output without inversion, but some use complementary channel with inversion.
I just assumed that Active-High is the default polarity we want everywhere and followed the code used in `pwmOCConfig(...)`. It also accepts output` as a parameter, although it was always passed from `timerHardware->output` before my changes.